pulsar-broker/src/test/java/org/apache/pulsar/broker/web/WebServiceTest.java

```diff
@@ -416,6 +430,7 @@ private void setupEnv(boolean enableFilter, String minApiVersion, boolean allowU
         config.setAdvertisedAddress("localhost"); // TLS certificate expects localhost
         config.setMetadataStoreUrl("zk:localhost:2181");
         config.setHttpMaxRequestSize(10 * 1024);
+        config.setEnableCompressMetricsData(true);
```

Comment:
This will change all the tests in this class run with compression enabled, and no tests to cover the case of without compression